app/javascript/components/widgets/traffic/panel.jsx

```diff
@@ -117,9 +117,9 @@ const Traffic = ({ cityName, startTimer }) => (
             </Column>
           </Row>
           <Notice>Images from:</Notice>
-          {getUniqUrls(trafficCams).map(url => (
-            <Notice key={url}>{url}</Notice>
-          ))}
+          {getUniqUrls(trafficCams).map(url => {
+            return url.includes('https') && <Notice key={url}>{url}</Notice>;
```

Comment:
I should have been clearer in the ticket. Maybe we can get rid of the schema portion and just show the domain?

Reply:
maybe try 
```js
new URL(url).hostname
```
https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

Reply:
ok